src/SqlTemplates/scd4.php

```diff
@@ -42,11 +42,11 @@
 SELECT \${snap_primary_key_lower}
           ,\${snap_table_cols}
           ,\${snap_default_cols}
-FROM last_curr_records
+FROM "last_curr_records"
```

Comment:
pridal som tu uvodzovky, bez nich to SNFK prevedie na UPPER_CASE a IMHO preto nesiel ten OM, ... niekde boli, niekde nie -> teraz je to zjednotene.